File: src/test/samples/sample.cpp
#include <iostream>

//#region FirstRegion
int x = 42;
//#endregion

// #region SecondRegion
class MyClass {
    //  #region    InnerRegion    
    void myMethod() {}
    //   #endregion    ends InnerRegion  

    // #region
    void myMethod2() {}
    //#endregion
};
//  #endregion

File: src/test/samples/sample.swift
import Foundation

//#region FirstRegion
let x = 42
//#endregion

//  #region SecondRegion
class MyClass {
    // #region     InnerRegion
    func myMethod() {}
    //   #endregion   ends InnerRegion

    // #region     
    func myMethod2() {}
    // #endregion   
}
//   #endregion
